scripts/mvftoms.py

```diff
@@ -428,11 +428,6 @@ def _cp_index(a1, a2, pol):
                        dumps=slice(options.quack, None))
 
         # The first step is to copy the blank template MS to our desired output
```

Comment:
The blank template MS was last used in 2017 (and last seen in 2018), so this is an ancient wart indeed :joy: Please chop this too.